README.md

```diff
@@ -168,8 +168,8 @@ AggregateError [ECONNREFUSED]:
 ```
 
 ### Broken TypeScript config file
-This boilerplate does not use TypeScript, but Vite which is used in the `app` package has built in support for TypeScript.
-This means that Vite (or rather `esbuild` which is used internally) will look for a TypScript config file (`tsconfig.json`) in the current folder and all parent folders until it finds one.
+This boilerplate does not use TypeScript, but Next which is used in the `app` package has built in support for TypeScript.
+This means that Next (or rather `esbuild` which is used internally) will look for a TypScript config file (`tsconfig.json`) in the current folder and all parent folders until it finds one.
```

Comment:
I know it wasn't your typo but while you're here can you fix "TypScript" please 😅 🙏